streams/src/main/java/org/apache/kafka/streams/state/internals/Segments.java

```diff
@@ -167,7 +172,11 @@ private void cleanup(final long segmentId) {
             if (segment != null && segment.id <= oldestSegmentId) {
                 segments.remove(segmentEntry.getKey());
                 segment.close();
-                segment.destroy();
+                try {
+                    segment.destroy();
+                } catch (IOException e) {
+                    log.error("Error destroying segment id {}", segment.id, e);
```

Comment:
Good point.  It looks like `RocksDBStore` already has a `toString` method.  So I will just rely on that.